src/Avalonia.Visuals/Size.cs

```diff
@@ -61,6 +61,15 @@ public Size(double width, double height)
         /// </summary>
         public double Height => _height;
 
+        /// <summary>
+        ///  Implicit convert from tuple of two double (double,double) to <see cref="Size"/>.
+        /// </summary>
+        /// <param name="value">tuple of two <see cref="double"/></param>
+        public static implicit operator Size((double width, double height) value)
```

Comment:
Adding implicit operators to/from tuples was already discussed in https://github.com/AvaloniaUI/Avalonia/pull/4589 and we decided to not implement for the moment. If we wanted to revisit that decision it should be done in an issue/PR not in an unrelated PR.